{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <font color = \"purple\">Linear regression, Ridge regression </font>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.preprocessing import StandardScaler, OrdinalEncoder, OneHotEncoder\n",
    "from sklearn.pipeline import Pipeline, make_pipeline\n",
    "from sklearn.model_selection import train_test_split,RandomizedSearchCV\n",
    "from sklearn.impute import SimpleImputer\n",
    "from sklearn.compose import ColumnTransformer, make_column_transformer, TransformedTargetRegressor\n",
    "from sklearn.linear_model import LinearRegression, Ridge, RidgeCV\n",
    "from sklearn.metrics import mean_squared_error, r2_score, make_scorer\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "from sklearn import metrics\n",
    "from sklearn.neighbors import KNeighborsRegressor\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "customers = pd.read_csv(\"Ecommerce Customers\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Email</th>\n",
       "      <th>Address</th>\n",
       "      <th>Avatar</th>\n",
       "      <th>Avg. Session Length</th>\n",
       "      <th>Time on App</th>\n",
       "      <th>Time on Website</th>\n",
       "      <th>Length of Membership</th>\n",
       "      <th>Yearly Amount Spent</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>mstephenson@fernandez.com</td>\n",
       "      <td>835 Frank Tunnel\\nWrightmouth, MI 82180-9605</td>\n",
       "      <td>Violet</td>\n",
       "      <td>34.497268</td>\n",
       "      <td>12.655651</td>\n",
       "      <td>39.577668</td>\n",
       "      <td>4.082621</td>\n",
       "      <td>587.951054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>hduke@hotmail.com</td>\n",
       "      <td>4547 Archer Common\\nDiazchester, CA 06566-8576</td>\n",
       "      <td>DarkGreen</td>\n",
       "      <td>31.926272</td>\n",
       "      <td>11.109461</td>\n",
       "      <td>37.268959</td>\n",
       "      <td>2.664034</td>\n",
       "      <td>392.204933</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>pallen@yahoo.com</td>\n",
       "      <td>24645 Valerie Unions Suite 582\\nCobbborough, D...</td>\n",
       "      <td>Bisque</td>\n",
       "      <td>33.000915</td>\n",
       "      <td>11.330278</td>\n",
       "      <td>37.110597</td>\n",
       "      <td>4.104543</td>\n",
       "      <td>487.547505</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>riverarebecca@gmail.com</td>\n",
       "      <td>1414 David Throughway\\nPort Jason, OH 22070-1220</td>\n",
       "      <td>SaddleBrown</td>\n",
       "      <td>34.305557</td>\n",
       "      <td>13.717514</td>\n",
       "      <td>36.721283</td>\n",
       "      <td>3.120179</td>\n",
       "      <td>581.852344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>mstephens@davidson-herman.com</td>\n",
       "      <td>14023 Rodriguez Passage\\nPort Jacobville, PR 3...</td>\n",
       "      <td>MediumAquaMarine</td>\n",
       "      <td>33.330673</td>\n",
       "      <td>12.795189</td>\n",
       "      <td>37.536653</td>\n",
       "      <td>4.446308</td>\n",
       "      <td>599.406092</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           Email  \\\n",
       "0      mstephenson@fernandez.com   \n",
       "1              hduke@hotmail.com   \n",
       "2               pallen@yahoo.com   \n",
       "3        riverarebecca@gmail.com   \n",
       "4  mstephens@davidson-herman.com   \n",
       "\n",
       "                                             Address            Avatar  \\\n",
       "0       835 Frank Tunnel\\nWrightmouth, MI 82180-9605            Violet   \n",
       "1     4547 Archer Common\\nDiazchester, CA 06566-8576         DarkGreen   \n",
       "2  24645 Valerie Unions Suite 582\\nCobbborough, D...            Bisque   \n",
       "3   1414 David Throughway\\nPort Jason, OH 22070-1220       SaddleBrown   \n",
       "4  14023 Rodriguez Passage\\nPort Jacobville, PR 3...  MediumAquaMarine   \n",
       "\n",
       "   Avg. Session Length  Time on App  Time on Website  Length of Membership  \\\n",
       "0            34.497268    12.655651        39.577668              4.082621   \n",
       "1            31.926272    11.109461        37.268959              2.664034   \n",
       "2            33.000915    11.330278        37.110597              4.104543   \n",
       "3            34.305557    13.717514        36.721283              3.120179   \n",
       "4            33.330673    12.795189        37.536653              4.446308   \n",
       "\n",
       "   Yearly Amount Spent  \n",
       "0           587.951054  \n",
       "1           392.204933  \n",
       "2           487.547505  \n",
       "3           581.852344  \n",
       "4           599.406092  "
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "customers.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Avg. Session Length</th>\n",
       "      <th>Time on App</th>\n",
       "      <th>Time on Website</th>\n",
       "      <th>Length of Membership</th>\n",
       "      <th>Yearly Amount Spent</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>count</th>\n",
       "      <td>500.000000</td>\n",
       "      <td>500.000000</td>\n",
       "      <td>500.000000</td>\n",
       "      <td>500.000000</td>\n",
       "      <td>500.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mean</th>\n",
       "      <td>33.053194</td>\n",
       "      <td>12.052488</td>\n",
       "      <td>37.060445</td>\n",
       "      <td>3.533462</td>\n",
       "      <td>499.314038</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>std</th>\n",
       "      <td>0.992563</td>\n",
       "      <td>0.994216</td>\n",
       "      <td>1.010489</td>\n",
       "      <td>0.999278</td>\n",
       "      <td>79.314782</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>min</th>\n",
       "      <td>29.532429</td>\n",
       "      <td>8.508152</td>\n",
       "      <td>33.913847</td>\n",
       "      <td>0.269901</td>\n",
       "      <td>256.670582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25%</th>\n",
       "      <td>32.341822</td>\n",
       "      <td>11.388153</td>\n",
       "      <td>36.349257</td>\n",
       "      <td>2.930450</td>\n",
       "      <td>445.038277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50%</th>\n",
       "      <td>33.082008</td>\n",
       "      <td>11.983231</td>\n",
       "      <td>37.069367</td>\n",
       "      <td>3.533975</td>\n",
       "      <td>498.887875</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75%</th>\n",
       "      <td>33.711985</td>\n",
       "      <td>12.753850</td>\n",
       "      <td>37.716432</td>\n",
       "      <td>4.126502</td>\n",
       "      <td>549.313828</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>max</th>\n",
       "      <td>36.139662</td>\n",
       "      <td>15.126994</td>\n",
       "      <td>40.005182</td>\n",
       "      <td>6.922689</td>\n",
       "      <td>765.518462</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Avg. Session Length  Time on App  Time on Website  \\\n",
       "count           500.000000   500.000000       500.000000   \n",
       "mean             33.053194    12.052488        37.060445   \n",
       "std               0.992563     0.994216         1.010489   \n",
       "min              29.532429     8.508152        33.913847   \n",
       "25%              32.341822    11.388153        36.349257   \n",
       "50%              33.082008    11.983231        37.069367   \n",
       "75%              33.711985    12.753850        37.716432   \n",
       "max              36.139662    15.126994        40.005182   \n",
       "\n",
       "       Length of Membership  Yearly Amount Spent  \n",
       "count            500.000000           500.000000  \n",
       "mean               3.533462           499.314038  \n",
       "std                0.999278            79.314782  \n",
       "min                0.269901           256.670582  \n",
       "25%                2.930450           445.038277  \n",
       "50%                3.533975           498.887875  \n",
       "75%                4.126502           549.313828  \n",
       "max                6.922689           765.518462  "
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "customers.describe()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'pandas.core.frame.DataFrame'>\n",
      "RangeIndex: 500 entries, 0 to 499\n",
      "Data columns (total 8 columns):\n",
      " #   Column                Non-Null Count  Dtype  \n",
      "---  ------                --------------  -----  \n",
      " 0   Email                 500 non-null    object \n",
      " 1   Address               500 non-null    object \n",
      " 2   Avatar                500 non-null    object \n",
      " 3   Avg. Session Length   500 non-null    float64\n",
      " 4   Time on App           500 non-null    float64\n",
      " 5   Time on Website       500 non-null    float64\n",
      " 6   Length of Membership  500 non-null    float64\n",
      " 7   Yearly Amount Spent   500 non-null    float64\n",
      "dtypes: float64(5), object(3)\n",
      "memory usage: 31.4+ KB\n"
     ]
    }
   ],
   "source": [
    "customers.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "y = customers['Yearly Amount Spent']\n",
    "X = customers[['Avg. Session Length', 'Time on App','Time on Website', 'Length of Membership']]\n",
    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=101)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "numeric_features  = [\"Avg. Session Length\" ,\"Time on App\" , \"Time on Website\" , \"Length of Membership\"]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(steps=[('simpleimputer', SimpleImputer(strategy='median')),\n",
       "                ('standardscaler', StandardScaler())])"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "numeric_preprocessing = make_pipeline(SimpleImputer(strategy='median'), \n",
    "                                      StandardScaler())\n",
    "numeric_preprocessing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "preprocessing = ColumnTransformer([\n",
    "    ('numeric', numeric_preprocessing, numeric_features)\n",
    "])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "preprocessing.fit(X_train);"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Avg. Session Length</th>\n",
       "      <th>Time on App</th>\n",
       "      <th>Time on Website</th>\n",
       "      <th>Length of Membership</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>-1.499941</td>\n",
       "      <td>-0.730834</td>\n",
       "      <td>-0.042984</td>\n",
       "      <td>0.274430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>-1.163743</td>\n",
       "      <td>2.110040</td>\n",
       "      <td>-0.060729</td>\n",
       "      <td>0.200101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>0.228480</td>\n",
       "      <td>-0.546783</td>\n",
       "      <td>0.048110</td>\n",
       "      <td>1.141258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>0.846567</td>\n",
       "      <td>0.508317</td>\n",
       "      <td>0.072683</td>\n",
       "      <td>-0.887289</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>-0.004191</td>\n",
       "      <td>0.494168</td>\n",
       "      <td>0.580927</td>\n",
       "      <td>0.518859</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>-0.238887</td>\n",
       "      <td>-0.383563</td>\n",
       "      <td>0.336960</td>\n",
       "      <td>-0.129222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>0.187526</td>\n",
       "      <td>-1.088966</td>\n",
       "      <td>1.400968</td>\n",
       "      <td>0.714731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>-1.198424</td>\n",
       "      <td>0.448846</td>\n",
       "      <td>0.357796</td>\n",
       "      <td>-0.577043</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0.848145</td>\n",
       "      <td>-0.473301</td>\n",
       "      <td>0.006833</td>\n",
       "      <td>0.174638</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>-0.837407</td>\n",
       "      <td>-0.681652</td>\n",
       "      <td>1.148571</td>\n",
       "      <td>1.289335</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>350 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Avg. Session Length  Time on App  Time on Website  Length of Membership\n",
       "202            -1.499941    -0.730834        -0.042984              0.274430\n",
       "428            -1.163743     2.110040        -0.060729              0.200101\n",
       "392             0.228480    -0.546783         0.048110              1.141258\n",
       "86              0.846567     0.508317         0.072683             -0.887289\n",
       "443            -0.004191     0.494168         0.580927              0.518859\n",
       "..                   ...          ...              ...                   ...\n",
       "63             -0.238887    -0.383563         0.336960             -0.129222\n",
       "326             0.187526    -1.088966         1.400968              0.714731\n",
       "337            -1.198424     0.448846         0.357796             -0.577043\n",
       "11              0.848145    -0.473301         0.006833              0.174638\n",
       "351            -0.837407    -0.681652         1.148571              1.289335\n",
       "\n",
       "[350 rows x 4 columns]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_columns = numeric_features \n",
    "X_train_enc = pd.DataFrame(preprocessing.transform(X_train), index=X_train.index, columns=new_columns)\n",
    "X_train_enc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "lin_rg = make_pipeline(preprocessing, LinearRegression())\n",
    "lin_rg.fit(X_train_enc, y_train);\n",
    "predictions= lin_rg.predict(X_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MAE: 1522.2502419256443\n",
      "MSE: 2317334.860754925\n",
      "RMSE: 1522.2794949531853\n"
     ]
    }
   ],
   "source": [
    "print('MAE:', metrics.mean_absolute_error(y_test, predictions))\n",
    "print('MSE:', metrics.mean_squared_error(y_test, predictions))\n",
    "print('RMSE:', np.sqrt(metrics.mean_squared_error(y_test, predictions)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <font color = \"purple\"> Ridge regression</font>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "ridge = Ridge(max_iter=200)\n",
    "pipe = Pipeline([(\"preprocessing\" , preprocessing), (\"ridge\" , ridge)])\n",
    "param_choices = {\n",
    "    \"ridge__alpha\" : np.arange(0,500)\n",
    "}\n",
    "random_search = RandomizedSearchCV(pipe, param_choices,\n",
    "                                   n_iter = 50, \n",
    "                                   verbose = 1,\n",
    "                                   random_state = 123)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fitting 5 folds for each of 50 candidates, totalling 250 fits\n"
     ]
    }
   ],
   "source": [
    "random_search.fit(X_train, y_train);"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'ridge__alpha': 5}"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "random_search.best_params_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.9797291636667446"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "random_search.best_score_"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# <font color = \"purple\"> KNeighbor regression</font>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "knnreg = KNeighborsRegressor()\n",
    "pipe = Pipeline([(\"preprocessing\" , preprocessing), (\"knnreg\" , knnreg)])\n",
    "param_choices = {\n",
    "    \"knnreg__n_neighbors\" : np.arange(1,500)\n",
    "}\n",
    "random_search2 = RandomizedSearchCV(pipe, param_choices,\n",
    "                                   n_iter = 50, \n",
    "                                   verbose = 1,\n",
    "                                   random_state = 123)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fitting 5 folds for each of 50 candidates, totalling 250 fits\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 310\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 310\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 310\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 310\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 310\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 424\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 424\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 424\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 424\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 424\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 289\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 289\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 289\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 289\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 289\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 420\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 420\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 420\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 420\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 420\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 416\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 416\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 416\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 416\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 416\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 285\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 285\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 285\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 285\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 285\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 489\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 489\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 489\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 489\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 489\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 336\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 336\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 336\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 336\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 336\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 472\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 472\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 472\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 472\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 472\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 389\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 389\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 389\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 389\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 389\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 439\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 439\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 439\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 439\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 439\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 494\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 494\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 494\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 494\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 494\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 309\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 309\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 309\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 309\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 309\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 470\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 470\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 470\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 470\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 470\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 479\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 479\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 479\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 479\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 479\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 286\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 286\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 286\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 286\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 286\n",
      "\n",
      "  UserWarning,\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 364\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 364\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 364\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 364\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py:687: UserWarning: Scoring failed. The score on this train-test partition for these parameters will be set to nan. Details: \n",
      "Traceback (most recent call last):\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_validation.py\", line 674, in _score\n",
      "    scores = scorer(estimator, X_test, y_test)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\metrics\\_scorer.py\", line 397, in _passthrough_scorer\n",
      "    return estimator.score(*args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\utils\\metaestimators.py\", line 120, in <lambda>\n",
      "    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\pipeline.py\", line 622, in score\n",
      "    return self.steps[-1][-1].score(Xt, y, **score_params)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\base.py\", line 553, in score\n",
      "    y_pred = self.predict(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_regression.py\", line 208, in predict\n",
      "    neigh_dist, neigh_ind = self.kneighbors(X)\n",
      "  File \"E:\\anacondajadid\\lib\\site-packages\\sklearn\\neighbors\\_base.py\", line 683, in kneighbors\n",
      "    (n_samples_fit, n_neighbors)\n",
      "ValueError: Expected n_neighbors <= n_samples,  but n_samples = 280, n_neighbors = 364\n",
      "\n",
      "  UserWarning,\n",
      "E:\\anacondajadid\\lib\\site-packages\\sklearn\\model_selection\\_search.py:921: UserWarning: One or more of the test scores are non-finite: [0.61047891        nan 0.68470987 0.23733616 0.48490446 0.86545477\n",
      " 0.52715109        nan        nan 0.56485198 0.75399785 0.01025691\n",
      " 0.34560081 0.84477094 0.61794676 0.38113156        nan 0.40882858\n",
      "        nan 0.54659712 0.300877          nan        nan 0.71664468\n",
      " 0.29242407        nan 0.14730534 0.76314129        nan 0.36960896\n",
      "        nan        nan 0.64738628 0.10752061        nan 0.31320987\n",
      " 0.76667063        nan 0.46633234 0.84653332        nan 0.58849447\n",
      "        nan        nan 0.23157892 0.332273   0.80097662        nan\n",
      " 0.35868067 0.4989676 ]\n",
      "  category=UserWarning\n"
     ]
    }
   ],
   "source": [
    "random_search2.fit(X_train, y_train);"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'knnreg__n_neighbors': 6}"
      ]
     },
     "execution_count": 83,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "random_search2.best_params_"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
